lib/helpers/is-interactive-element.js

```diff
@@ -2,9 +2,11 @@
 
 const ast = require('./ast-node-info');
 
+// Spec: https://html.spec.whatwg.org/#interactive-content
 const INTERACTIVE_TAG_NAMES = new Set([
+  'a',
```

Comment:
This is incorrect. The spec link pasted just above specifically says that `<a>` is only interactive with an `href`:

> a (if the href attribute is present)audio (if the controls attribute is present)buttondetailsembediframeimg (if the usemap attribute is present)input (if the type attribute is not in the Hidden state)labelobject (if the usemap attribute is present)selecttextareavideo (if the controls attribute is present)


<img width="983" alt="Screen Shot 2020-10-03 at 9 25 27 AM" src="https://user-images.githubusercontent.com/12637/94992819-7438b100-055a-11eb-8f63-edf2c23f1e2a.png">

Reply:
Also, we already have a different rule that enforces this: https://github.com/ember-template-lint/ember-template-lint/blob/master/docs/rule/link-href-attributes.md

To assistive technology, an `<a>` is an `<a>`, with or without the `href`.

Reply:
> this is one place where the ARIA spec supersedes the HTML spec.

Need supporting evidence here...

Reply:
1. our own rule that requires href on the a element (and the documentation that rule links to)
2. “In case of conflict, consider users over authors over implementers over specifiers over theoretical purity” from https://www.w3.org/TR/html-design-principles/#priority-of-constituencies